packages/connection-form/src/utils/read-preferences.ts

```diff
@@ -1,12 +1,18 @@
 import type { ReadPreferenceMode } from 'mongodb';
 import { ReadPreference as MongoReadPreference } from 'mongodb';
 
+export const defaultReadPreference = 'default';
+
 interface ReadPreference {
   title: string;
-  id: ReadPreferenceMode;
+  id: ReadPreferenceMode | typeof defaultReadPreference;
 }
 
 export const readPreferences: ReadPreference[] = [
+  {
+    title: 'Default',
+    id: defaultReadPreference,
```

Comment:
Nit: do we need to overload `id` with the value? Considering that we are also only using this "type" in the advanced tab and is quite tightly coupled with the usage there, we could just delete this file and do this instead:

What about this:

```
const readPreferences = [{
   id: 'default',
   title: 'Default',
   value: ''
}, ...]
```

the code in the other file will be much simpler as the `RadioBox` should handle everything. The only thing still needed will be `value={readPreference ?? ''}` on the `RadioBoxGroup`.

Reply:
Updated. Thanks for the suggestion, definitely reduces, sometimes with smaller changes like this its easy to just do it without thinking of a more simple way of implementing it.